data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1875 @@ mission "Stone of our Fathers 5"
+			`	You look up one last desperate time, hoping to see Hjlod coming to rescue you. But instead you see a grey smudge on the horizon. A distant person who has long left you for dead.`
+			`	Alone now with the Ice-Slug and the pain. You consider briefly your mistakes before you are dissolved and eaten.`
+				die
+			label help
+			`	Damn your pride, "Hjlod help! Something's got my foot!"`
+			`	Hjlod looks back at you with a dull expression on her face. "Is it purple-grey in color?"`
+			`	"Yes, yes! Quick help. I don't know what to do."`
+			`	"Vhat you do, is listen to Hjlod, and not make stompy circles in snow like spoilt child."`
+			`	"Please Hjlod!" You cry as the last of your boot dissolves away and you feel the slimy slug on your exposed foot.`
+			`	"It is too late. Ice-Slug has you. Vhat you do now is die. Horribly." Hjlod turns and starts jogging away from you.`
+			choice
+				`	(Try and pull your leg free.)`
+					goto free
+			label travel
+			`	The cold monotonous trudging weighs on you as the hours pass. You're dimly aware of Nifel's two suns as they make their journey across the wide open sky.`
+			`	Evening and exhaustion approach hand in hand and as the last sun sets you're not sure how much farther you can go. `
+			`	"There!" Hjlod breaks the silence of hours as she points at the weak flicker of light in the twilight landscape.`
+			`	"This is good." Hjlod admits hoarsely as you both stumble toward the light. "Ve Vould have not survived night."`
+			` 	The night air grows too cold. You slow as your body begins to shutdown and frostbite gnaws at your hands, feet and exposed arm. The light is still too distant, and you can barely move. You think you see the shadows of humanoid figures against the light. But then you stumble and fall. You crawl, for as long as you can, you crawl. And then nothing.`
+			choice
+				`	(In the darkness you feel hands, and then warmth, and then more nothing.)`
+					goto village
+			label village
+			`	The pale morning light of the first sun filters in through a synthetic window and onto your face.`
+			`	Your body aches and you try to sink back into sleep, but something feels off and you focus on your surroundings.`
+			`	You're in a half-wooden, half-synthetic barracks. A strange mix of primitive earth-tech and old colonial architecture.`
+			`	That, in of itself, isn't that alarming. What is, however, is the group of people gathered around your bed quietly staring and pointing at you.`
+			choice
+				`	"Excuse me?"`
+			`	The crowd steps back and an elderly woman steps forward. Her grey hair is rolled up into tight elegant braids, and her face and hands are covered in faded blue tattoos.`
```

Comment:
It's this way because its the only way I know how to do it. This is my 2nd PR and I don't have a strong familiarity with the formatting stuff. I can't make act on your suggestion here, because I don't know how to atm.

Reply:
I understand, if I get some time I'll maybe see if I can work it out.

Reply:
Terin has implemented the looped dialog options. I think this is now resolved.

Reply:
> Thanks to Terin, I think both issues have now been resolved.

Terin dealt with the looping conversation options, but there's still at least one major block repeated.

Reply:
Which block? There are some subtle differences between a few of the options depending on how you get to the village, if that is what you're thinking.

Reply:
Yeah, I don't think there's enough there to break out into separate blocks, unless we REALLY go line by line.